react/components/OrderInfo/OrderOptions.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FunctionComponent } from 'react'
+import { InjectedIntlProps, injectIntl } from 'react-intl'
+import ButtonLink from '../ButtonLink'
+
+interface Props {
+  allowCancellation: boolean
+  takeaway?: boolean
+  className?: string
+  fullWidth?: boolean
+  orderId?: string
+}
+
+const OrderOptions: FunctionComponent<Props & InjectedIntlProps> = ({
+  allowCancellation,
+  takeaway,
+  intl,
+  className = '',
+  fullWidth,
+  orderId,
+}) => (
+  <div className={`${className} flex flex-wrap justify-center flex-nowrap-m`}>
+    <div className="mr5-ns mb5-s mb0-m w-100 w-auto-m">
+      {takeaway ? (
+        <ButtonLink variation="secondary" fullWidth={fullWidth} url="">
```

Comment:
Empty url?

Reply:
This is because of inStore integration, not completed yet.